logstash-core/lib/logstash/filter_delegator.rb

```diff
@@ -37,17 +39,19 @@ def config_name
     end
 
     def multi_filter(events)
-      @metric_events.increment(:in, events.size)
+      @metric_events_in.increment(events.size)
 
-      clock = @metric_events.time(:duration_in_millis)
+      start_time = java.lang.System.current_time_millis
       new_events = @filter.multi_filter(events)
-      clock.stop
+      @metric_events.report_time(
```

Comment:
@jakelandis ah gnarf I just noticed I rebased wrong :( no need to find all instances of these, none of them were fixed. Funny ... we can be even faster it seems sec :D